packages/lingua-franca/gatsby-browser.js

```diff
@@ -1,11 +1,6 @@
 // This hooks ups client-side app analytics
 // it's based on how the google analytics plugin works for gatsby
 // https://github.com/gatsbyjs/gatsby/blob/master/packages/gatsby-plugin-google-analytics/src/gatsby-browser.js
-const { setInitialTargetLanguage } = require("./src/lib/setInitialTargetLanguage")
-
-exports.onClientEntry = () => {
-  window.onload = () => { setInitialTargetLanguage() }
-}
 
 exports.onRouteUpdate = ({ location, prevLocation }) => {
```

Comment:
@petervdonovan, could this be the cause for #90?

Reply:
That seems possible.

When I was doing #91 I realized that when this branch was merged, `setInitialTargetLanguage()` invocations appeared in several places. I had let this by because I did not (and still do not) really understand when Gatsby calls everything and where different setup processes should go (and because I did not do due diligence in reviewing my work before the merge).

I think the solution in #91 "should work" based on the [documentation](https://www.gatsbyjs.com/docs/reference/config-files/gatsby-browser/#onRouteUpdate). I do not understand how it would work to just have `setInitialTargetLanguage()` hooked onto `onClientEntry` because the target language should be set whenever the user navigates to a different page, but given the other places where `setInitialTargetLanguage()` was invoked, maybe they were collectively sufficient.